app/Http/Controllers/UserController.php

```diff
@@ -189,4 +191,9 @@ public function selectbatch(Request $request)
 
         return $batches;
     }
+
+    public function currentUser(Request $request)
+    {
+        return  new UserResource(Auth::user());
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.LanguageConstructSpacing.IncorrectSingle</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 197</br>Language constructs must be followed by a single space; expected 1 space but found "··"